src/styles/components/list-organizations.styl

```diff
@@ -0,0 +1,81 @@
+.list
```

Comment:
this needs to be more specific. maybe `listOrgs` or `list-orgs`? I don't think BEM should prevent the use of dashes within block, element or modifier descriptions (which for a modifier is two dashes `--`), but could see how it might look confusing.